File: Data Structures & Algorithms/longest-common-prefix/submission-0.cs
public class Solution {
    public string LongestCommonPrefix(string[] strs) {
        if (strs == null || strs.Length == 0){
            return "";
        }

        for (int i = 0; i < strs[0].Length; i++){
            for (int j = 1; j < strs.Length; j++){
                if (i == strs[j].Length || strs[0][i] != strs[j][i]){
                    Console.WriteLine(strs[0]);
                    return strs[0][0..i];
                }
            }
        }
        return strs[0];
    }
}

File: Data Structures & Algorithms/anagram-groups/submission-0.py
class Solution:
    def groupAnagrams(self, strs: List[str]) -> List[List[str]]:
        hmap = defaultdict(list)
        for s in strs:
            count = [0]*26
            for c in s:
                code = ord(c) - ord('a')
                count[code] += 1
            hmap[tuple(count)].append(s)
        return hmap.values()